src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,46 @@
+
+.calendar {
+  position: relative;
+  top: 50%;
+  left: 50%;
```

Comment:
Let's move these styles to page__calendar class.

Reply:
It's a separate BEM block and in order to be reusable it can't be a page element